### DISCUSSION TITLE
Sponsor/donation -- where?

### DISCUSSION THREAD

**Discussion by WilliamDEdwards** [pydantic/pydantic#4631] [Question] [answered]
Hi,

https://pydantic-docs.helpmanual.io/blog/pydantic-v2/#plan-timeframe says:

> if you work for a large company who use pydantic a lot, you might encourage the company to sponsor me a meaningful amount

However, there is no further mention of where and how to donate or sponsor. Searching 'donation' and 'sponsor' on https://pydantic-docs.helpmanual.io/ does not yield any results either.

Am I missing something?

**Comment by samuelcolvin** [comment, marked as answer]
> Am I missing something?

Kind of, :smile:, there's a "Sponsor" button a the top of this page.

But thanks for the suggestion, I'll add a link to the docs when updating the docs for V2.

**Reply by WilliamDEdwards** [reply]
Yes, it would be good to mention that GitHub is used :).

**Reply by samuelcolvin** [reply]
Will do, thanks so much for sponsoring me. 🙏🙏🙏

**Reply by black-snow** [reply]
So donations to pydantic (specifically) still go to https://github.com/sponsors/samuelcolvin in 2024? :)

**Reply by Viicos** [reply]
@black-snow there's also https://github.com/sponsors/pydantic.

**Reply by black-snow** [reply]
#facepalmEmoji, thanks
